### PR TITLE
Add the use cases for the platform implementer persona

### PR DESCRIPTION
**Pull request by ckemper67**
To address #165, this PR adds the use cases for the platform implementer persona. The use cases are interesting, because the platform implementer's responsibility is to make some of the other roles successful.

To highlight that relationship between the roles, I decided to bold the use of other personas in the use case description.

### CODE REVIEW

File: usecases.md [hunk 1]
```diff
@@ -160,8 +161,18 @@ impact of repeated request on the rest of the infrastructure.
...
+* As a platform implementer I can make my organization's **security administrators and compliance officers** successful.
+* As a platform implementer I can provide central administration of cloud resources to my **operators and administrators**.
+* As a platform implementer I can provide compartmentalization of cloud resources for delegation purposes by **operators and administrators**. 
+* As a platform implementer I can allow my **operators and administrators** to delegate administration of resources.
+* As a platform implementer I can enable my **security administrators** to constrain the behavior of **developers** by setting guardrails.
+* As a platform implementer I can enable my **security administrators and compliance officers** to enforce auditing of access policies. 
+* As a platform implementer I can enable my **security administrators and compliance officers** the auditing of resource access.
+* As a platform implementer I can enable my **compliance officers** the setting of audit logging policies for my organization's resources
+* As a platform implementer I can enable my **compliance officers** to certify the non-violation of my organization's compliance needs.
```
Comment (lumjjb) [inline usecases.md]: nit: spacing between bullets for consistency

File: usecases.md [hunk 2]
```diff
@@ -160,8 +161,18 @@ impact of repeated request on the rest of the infrastructure.
 
 * As an end user, I can understand the effect of changes to policy that I am making
 
-
-
+# Platform Implementer
+
+* As a platform implementer I can make my organization's **enterprise, network and quota operators** successful.
+* As a platform implementer I can make my organization's **security administrators and compliance officers** successful.
```
Comment (lumjjb) [inline usecases.md]: These two points seem to be a summary of the below points. Organizing it into the two groups could be very helpful for readability.

File: usecases.md (hunk 1 above)
Comment (lumjjb) [inline usecases.md]: My assumption of a platform implementer is that he/she should be responsible across multi-cloud or hybrid cloud boundaries - like in the security administrator case. Or is the platform implementer here scoped to a single cloud?

Comment (ckemper67) [reply usecases.md]: No I think the multi-cloud and hybrid deployments are actually very relevant to this persona, maybe to all personas here. That might be something to put into the intro paragraph.

File: usecases.md (hunk 2 above)
Comment (ckemper67) [reply usecases.md]: Do you mean one group each for operators and administrators or one group each for summary and details?

File: usecases.md (hunk 1 above)
Comment (ckemper67) [reply usecases.md]: ack

Comment (ckemper67) [reply usecases.md]: I decided to add a dedicated intro paragraph for the platform implementer persona and specifically mention the bridging of heterogeneous platforms.

File: usecases.md (hunk 2 above)
Comment (lumjjb) [reply usecases.md]: one group each for operators and administrators

**Comment by lumjjb** [comment]
@pragashj any thoughts on the addition?

**Comment by lumjjb** [comment]
Merging as discussed on 09/18 call.

**Pull request merged**